navigation/presentation.html

```diff
@@ -2,43 +2,1222 @@
+              <br />
+              Using gestural navigation helps to create a journey between different scenes within an app and gestures include touching and dragging, both horizontally and vertically, as well as zooming in and out. It’s highly immersive and interactive, creating a dynamic experience.
+              <br /><br />
+              The great thing about this pattern is that it’s relatively simple to grasp for even the most inexperienced user as the gestures are often intuitive and only require a little experimentation to get right.
+              <br /><br />
+              But if the navigation is cluttered, then without proper visual hints and cues, users can become confused about how to interact with the app.
+              <br />
+              This will increase the effort of the user.
+            </aside>
+          </section>
+          <!-- 3.6 slide -->
+          <section class="bullets-and-picture-slide">
+            <h4 class="slide-name">3D touch</h4>
+            <div>
+              <img src="assets/images/3d_touch.jpg" alt="3d touch" />
+              <div class="pros-cons-section">
+                <p class="big">
+                  Pros:
+                </p>
+                <ul>
+                  <li>
+                    save users a lot of time by skipping several taps within an app
+                  </li>
+                  <li>
+                    enables quick, lightweight interaction
+                  </li>
+                </ul>
+                <p class="big">
+                  Cons:
+                </p>
+                <ul>
+                  <li>
+                    users have to remember which apps have quick actions
+                  </li>
+                </ul>
+              </div>
+            </div>
+
+            <aside class="notes">
+              3D Touch is a feature, which triggers a small submenu you can use to run specific commands.
+              All you have to do is to press hard on an app icon.
+              <br /><br />
+              It was first introduced by Apple. Later, a similar feature appeared on android called "App Shortcuts" and allows you to have an experience similar to "3D Touch".
+              <br /><br />
+              It’s a way to create a navigation shortcut, showing the key actions for the selected app.
+              Another common use for this mobile navigation is the preview of content.
+              <br /><br />
+              Some popular apps that support 3D Touch include Amazon, Chrome, Gmail, Google Maps, Instagram, Netflix, Twitter, Uber, WhatsApp, YouTube, etc.
+            </aside>
+          </section>
+        </section>
+        <!-- 4 slide -->
+        <section class="slide-with-text-and-logos">
+          <h4 class="slide-name">Navigation libraries</h4>
+          <div>
+            <div>
+              <p><b>React Navigation: </b>well documented, actual, expo, community, JS</p>
```

Comment:
this still reads strangely like "expo" is an adjective.
Maybe it's better to make it like "well documented, actual, driven by community, written in JS, supports Expo"
(sorry for too much nerdiness)

Reply:
fixed